druid/src/widget/split.rs

```diff
@@ -91,6 +93,11 @@ impl<T> Split<T> {
         self.draggable = draggable;
         self
     }
+    /// Set whether the splitter handle is a solid rectangle or two parallel lines.
+    pub fn solid(mut self, solid: bool) -> Self {
```

Comment:
The over-the-top thing to do here would be to have a  `SplitterStyle` enum but I don think that's necessary, however I might just give this a slightly more descriptive name; something like `draw_solid_splitter` or `solid_splitter`  or ??

Reply:
I've been stewing for a while on wanting to offer more appearance options here and more generally. I think an enum adds a small amount of flexibility, but is kinda constraining.. But it's the obvious choice if we go beyond two appearance styles.
Something I was kicking around was a Fn that gets the paint ctx, start and end coordinates (of the bar centerline) and width as set. Then the expectation is that you draw within the rectangle this provides, however you like.
It's possible we could use a hidden inner widget for this, if we want to leverage the built-in clipping ability and coordinate space?

Reply:
To be clear: I still wouldn't be opposed to having one or several built-in options for this.

Reply:
I started out with exactly that design, but it seemed a bit overkill. Besides the solid and stroked styles we have, is there another type of splitter that is in common use at the moment? Windows 10, macOS, and GNOME 3 all use a solid filled splitter.

KDE and some versions of GNOME 3 have sort of 3D embossed splitters, but IMO something like that would look out of place in Druid at the moment. We could reconsider this if/when Druid has more ways of styling all the other widgets.

(Now that I think of it, an ideal way of handling the styling of the splitter handle could be to set some values on the enclosing `Env`. We draw a solid rectangle as the splitter in all cases, and let the user set custom values on the `Env` for its stroke and fill properties. But I do feel our current method is enough for now.)

As for the name, how does `fill_splitter_handle` sound? Nice and verbose, and fits nicely into this snippet from my app:

```rust
Split::vertical(make_sidebar(), make_todo_list())
        .draggable(true)
        .split_point(0.3)
        .splitter_size(4.0)
        .min_size(160.0)
        .fill_splitter_handle(true)
```

Reply:
this sounds good to me!